### DISCUSSION TITLE
Support Colbert rerank

### DISCUSSION THREAD

**Discussion by douxiaofeng99** [milvus-io/milvus#34943] [Ideas & Feature requests]
Colbert V2 has significantly precise for reranking, which ,milvus version will support this?

**Comment by xiaofan-luan** [comment]
We are actually doubting on this. 
We may support colbert for retrieval but for ranking why not simply use a cross encoder?

**Reply by douxiaofeng99** [reply]
实验发现使用这种算法做重排效果很好。或者在重排时，是否可以自定义或者调用别的重排模型，谢谢

**Reply by xiaofan-luan** [reply]
is there a comparison between colbert and other rankers like t5?

By saying supporting colbert, what is the desired parttern? did you host colbertV2 model yourself and store token embeddings into milvus or you want milvus to do the embedding generation?
